Issue against Django `master`, especially Django Guardian

Running against Django `master` there are a **lot** of issues. 

```
1 warnings, 265 error in 137.92 seconds
```

Most of these are caused by Django Guardian. 

Excluding that from the requirements leads to something more reasonable: 

```
5 failed, 239 passed, 1 warnings, 21 error in 11.27 seconds
```

Only two types of failure: 

```
tests/test_utils.py:153: in setUp
    self.get_model = rest_framework.utils.model_meta.models.get_model
E   AttributeError: module 'django.db.models' has no attribute 'get_model'

...

tests/browsable_api/test_browsable_api.py:36: in test_login_shown_when_logged_out
    self.assertContains(response, '>Log in<')
.tox/py35-djangomaster/lib/python3.5/site-packages/django/test/testcases.py:398: in assertContains
    msg_prefix + "Couldn't find %s in response" % text_repr)
E   AssertionError: False is not true : Couldn't find '>Log in<' in response
```

Errors are all these:

```
.tox/py35-djangomaster/lib/python3.5/site-packages/_pytest/assertion/rewrite.py:682:
...
E   TypeError: Call constructor takes either 0 or 3 positional arguments

...
tests/test_filters.py:11: in <module>
    from django.utils import unittest
E   ImportError: cannot import name 'unittest'
```

I don't think there's much work there but it's masked by the Guardian issues.

--- a/tox.ini
+++ b/tox.ini
@@ -39,3 +39,25 @@ commands = mkdocs build
 deps =
        -rrequirements/requirements-testing.txt
        -rrequirements/requirements-documentation.txt
+
+# Specify explicitly to exclude Django Guardian against Django master (various Pythons)
+[testenv:py27-djangomaster]
+deps =
+       https://github.com/django/django/archive/master.tar.gz
+       -rrequirements/requirements-testing.txt
+       markdown==2.5.2
+       django-filter==0.10.0
+[testenv:py34-djangomaster]
+deps =
+       https://github.com/django/django/archive/master.tar.gz
+       -rrequirements/requirements-testing.txt
+       markdown==2.5.2
+       django-filter==0.10.0
+
+[testenv:py35-djangomaster]
+deps =
+       https://github.com/django/django/archive/master.tar.gz
+       -rrequirements/requirements-testing.txt
+       markdown==2.5.2
+       django-filter==0.10.0
+

